vector/src/main/java/im/vector/util/VectorUtils.java

```diff
@@ -699,60 +693,37 @@ public static String getApplicationVersion(final Context context) {
-                    try {
-                        mMainAboutDialog.dismiss();
-                    } catch (Exception e) {
-                        Log.e(LOG_TAG, "## displayThirdPartyLicenses() : " + e.getMessage(), e);
-                    }
-                }
-                mMainAboutDialog = null;
+    private static void displayInWebView(final Context context, String url) {
+        WebView wv = new WebView(context);
+        wv.loadUrl(url);
+        wv.setWebViewClient(new WebViewClient() {
+            @Override
+            public boolean shouldOverrideUrlLoading(WebView view, String url) {
+                view.loadUrl(url);
+
+                return true;
             }
+        });
 
-            activity.runOnUiThread(new Runnable() {
-                @Override
-                public void run() {
-                    WebView view = (WebView) LayoutInflater.from(activity).inflate(R.layout.dialog_licenses, null);
-                    view.loadUrl("file:///android_asset/open_source_licenses.html");
-
-                    View titleView = LayoutInflater.from(activity).inflate(R.layout.dialog_licenses_header, null);
-
-                    view.setScrollbarFadingEnabled(false);
-                    mMainAboutDialog = new AlertDialog.Builder(activity)
-                            .setCustomTitle(titleView)
-                            .setView(view)
-                            .setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
-                                @Override
-                                public void onClick(DialogInterface dialog, int which) {
-                                    mMainAboutDialog = null;
-                                }
-                            })
-                            .setOnCancelListener(new DialogInterface.OnCancelListener() {
-                                @Override
-                                public void onCancel(DialogInterface dialog) {
-                                    mMainAboutDialog = null;
-                                }
-                            })
-                            .show();
-                }
-            });
-        }
+        new AlertDialog.Builder(context)
+                .setView(wv)
+                .setPositiveButton(android.R.string.ok, null)
+                .show();
     }
 
     /**
-     * Open a webview above the current activity.
+     * Open a webview above the current activity with a title.
      *
      * @param context the application context
      * @param url     the url to open
+     * @param title   title of the dialog
      */
-    private static void displayInWebview(final Context context, String url) {
+    private static void displayInTitleWebView(final Context context, String url, String title) {
         WebView wv = new WebView(context);
         wv.loadUrl(url);
         wv.setWebViewClient(new WebViewClient() {
```

Comment:
Maybe this one is not necessary as well.

Reply:
Removed both web view clients and didn't see any issues.

Reply:
ok, thanks